software/bedtools/genomecov/main.nf

```diff
@@ -0,0 +1,33 @@
+def MODULE = "bedtools_genomecov"
+params.publish_dir = MODULE
+params.publish_results = "default"
+
+process BEDTOOLS_GENOMECOV {
+    tag {bam}
+
+    publishDir "${params.out_dir}/${params.publish_dir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename ->
+                    if (params.publish_results == "none") null
+                    else filename }
+
+    //container "docker.pkg.github.com/nf-core/$MODULE"
```

Comment:
should subcommands share a container?

Reply:
This is something I also thought about, in these cases all the modules share the same container so I don't know if it makes sense to create a different container for each of them, maybe a single Conda and Docker file at `bedtools` or the equivalent could do it, although this will create two types of modules.